tips/glossary.md

```diff
@@ -14,6 +14,29 @@ window.addEventListener("click", function () {
 });
 ```
 
+### Frequently Used Terms Definitions 
+**git**: an open source, distributed version-control system
+
+**GitHub**: a platform for hosting and collaborating on Git repositories
+
+**commit**: a Git object, a snapshot of your entire repository compressed into a SHA
+
+**branch**: a lightweight movable pointer to a commit
+
+**clone**: a local version of a repository, including all commits and branches
+
+**remote**: a common repository on GitHub that all team member use to exchange their changes
+
+**fork**: a copy of a repository on GitHub owned by a different user
+
+**pull request**: a place to compare and discuss the differences introduced on a branch with reviews, comments, integrated
+tests, and more
+
+**HEAD**: representing your current working directory, the HEAD pointer can be moved to different branches, tags, or commits
+when using git checkout
+
 ### Additional Resources
 
 [Anonymous Function](https://en.wikipedia.org/wiki/Anonymous_function)
+
+[Frequently Used Terms Definitions](services@github.com)
```

Comment:
Why an email address?

Reply:
that's the source of the defined terms, maybe if someone wants to learn more about Github or have any other request or issue could email them. I can discard it if you think it's unnecessary

Reply:
I would add a website or something like this.

Reply:
https://docs.github.com/en/get-started/quickstart/github-glossary maybe it could help better

Reply:
yes like that one (y)